src/.gitignore

```diff
@@ -275,3 +275,6 @@ rard/media/
 .env
 .envs/*
 !.envs/.local/
+
+### dump.json
+dump.json
```

Comment:
Better safe than sorry but we might not need this anymore due to the changes to loaddata

Reply:
Yeah, Roger must have added this, but like you say: better safe than sorry